### PR TITLE
Fix #18 cookie-parser populates req.cookies (plural)

### LINKED ISSUE #18: cookie-parser populates req.cookies (plural)
`cookie-parser` populates `req.cookies` (plural) and `plugToRequest` incorrectly checks `req.cookie` (singular).

> Parse Cookie header and populate req.cookies with an object keyed by the cookie names.

https://github.com/expressjs/cookie-parser

```javascript
function plugToRequest(req, res) {
  if (req.cookie) {
    _rawCookie = req.cookie;
  } else if (req.headers && req.headers.cookie) {
    setRawCookie(req.headers.cookie);
  } else {
    _rawCookie = {};
  }

  _res = res;
}
```

https://github.com/eXon/react-cookie/blob/master/index.js#L67-L77

Not really breaking since `req.headers.cookie` is a fallback and that exists but I thought I would point it out.

### PR DESCRIPTION
**Pull request by pjvds**
This pull request fixes issue #18. It has test coverage and is still backwards compatible.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -67,6 +67,8 @@ function setRawCookie(rawCookie) {
 function plugToRequest(req, res) {
   if (req.cookie) {
     _rawCookie = req.cookie;
```
Comment (eXon) [inline index.js]: You're missing a closing bracket here.

**Comment by pjvds** [comment]
@eXon Thanks for the pointer. I pushed a fix for this mistake and the build is running fine now and the tests are green.

**Comment by pjvds** [comment]
@eXon Any reason why this one isn't pulled yet?

**Comment by eXon** [comment]
Thanks for the PR!

**Comment by pjvds** [comment]
Thanks for the pull!

**Pull request merged**